src/it/parent-2x/invoker.properties

```diff
@@ -1,4 +1,3 @@
 # install, not verify, because we want to check the artifact as we would be about to deploy it
-# !enforcer.fail to suppress (TODO do in plugin-pom/pom.xml via requirePluginVersions#unCheckedPluginList): Some plugins are missing valid versions:(LATEST RELEASE SNAPSHOT are not allowed ) \\ [INFO] org.jenkins-ci.tools:maven-hpi-plugin. The version currently in use is 1.xyz-SNAPSHOT
 # release.skipTests normally set in jenkins-release profile since release:perform would do the tests
-invoker.goals=-Pjenkins-release -Denforcer.fail=false -Drelease.skipTests=false clean install
+invoker.goals=-Pjenkins-release -Drelease.skipTests=false clean install
```

Comment:
Taking advantage of https://github.com/jenkinsci/plugin-pom/pull/61#discussion_r114632901.